src/main/java/game/RacingGame.java

```diff
@@ -0,0 +1,79 @@
+package game;
+
+import java.util.Random;
+
+public class RacingGame {
+
+    public final Integer POINT_FORWARD = POINT;
+
+    public final static Integer POINT = 4;
+
+    /**
+     * 자동차 경주 게임
+     * @param cars
+     * @param count
+     */
+    public void racing(int cars, int count){
```

Comment:
메소드는 동사형으로 시작되어야합니다 :)
```suggestion
    public void race(int cars, int count){
```

Reply:
```suggestion
    public void racing(int carCount, int count){
```
게임에 참가하는 자동차의 수 이므로 carCount 와 같은 이름이 좀 더 적절하지 않을까요~?